addons/survey/static/src/js/survey_breadcrumb.js

```diff
@@ -0,0 +1,46 @@
+odoo.define('survey.breadcrumb', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
+
+publicWidget.registry.SurveyBreadcrumbWidget = publicWidget.Widget.extend({
+    xmlDependencies: ['/survey/static/src/xml/survey_templates.xml'],
+    template: "survey.survey_breadcrumb_template",
+    events: {
+        'click .breadcrumb-item a': '_onBreadcrumbClick',
+    },
+
+    /**
+     * @override
+     */
+    init: function (parent, options) {
+        this._super.apply(this, arguments);
+        this.canGoBack = options.canGoBack;
```

Comment:
Can't we keep simply options in template and remove variable copy-pasting ?

Reply:
It's pretty usual to copy options to widget properties.
Check "slide_course_quiz_finish.js" for example.

Options are more things that are used to initialize the widget, while properties can change from the behavior of the widget (like "currentPageId" here).
(At least that's my opinion).